fmriprep/cli/run.py

```diff
@@ -786,7 +786,18 @@ def build_workflow(opts, retval):
     boilerplate = retval['workflow'].visit_desc()
 
     if boilerplate:
-        (logs_path / 'CITATION.md').write_text(boilerplate)
+        # To please git-annex users and also to guarantee consistency
+        # among different renderings of the same file, first remove any
+        # existing one
+        citation_files = {
+            ext: str(logs_path / 'CITATION.' + ext)
+            for ext in ('bib', 'tex', 'md', 'html')
+        }
+        for citation_file in citation_files.values():
+            if op.lexists(citation_file):
+                citation_file.unlink()
```

Comment:
This would not work if `citation_file` is a string (should be addressed by my above comment)

Reply:
rright... first I followed the scheme of using pathlib and then felt that this function seems to not really need it - majority of "operations" is just stringification of pathlib objects and then a single `write_text`... so I regressed to using strings.  So is this not desired and fmriprep in general should prefer pathlib over regular strings for clarity/uniformity?

Reply:
I generally prefer pathlib because it is pretty neat. Some functions expect `Path`s down the line. If you don't mind, please go ahead with Path.

Reply:
done. let's see if tests pass alright ;)